### ISSUE TITLE
Standardizer charges sulphur

### ISSUE THREAD

**Issue by sonial** [chembl/ChEMBL_Structure_Pipeline#34]
Hi,

I noticed the standardizer adds a charge to suphur, which doesn't look correct to me.
For instance
![image](https://user-images.githubusercontent.com/7021751/141513775-79c91159-374e-422b-8097-83cc3e4511b3.png)
is standardised to 
![image](https://user-images.githubusercontent.com/7021751/141513835-97c8c1db-fb42-4224-9d40-d542dd9509e1.png)
and 
![image](https://user-images.githubusercontent.com/7021751/141513926-164a267b-bf26-4aeb-a49d-45489f66dea4.png)
is standardised to 
![image](https://user-images.githubusercontent.com/7021751/141513969-df2e03f9-4994-4072-8dc9-f2b41365abb1.png)

**Comment by BZdrazil** [comment]
Hi,

Thanks for sending all the issues you find with the standardizer and sorry for the late reply.

Since it has been a while since you sent us the comment regarding the charges on the sulphur, we ran both structures through the standardizer again. It seems that the standardizer is standardizing thes tructures correctly as we had intended. I hope this resolves the issue.  

![Unbenannt](https://user-images.githubusercontent.com/51444479/211532840-f937c0fb-6353-4e27-9793-d42c15f3c0b4.png)
